### ISSUE TITLE
[Update Request]: npcap

### ISSUE THREAD

**Issue by DevAlsicJanDeclercq** [microsoft/winget-pkgs#143935]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Insecure.Npcap

### Package Version

1.79

### Please describe the changes you would like to see

https://npcap.com/dist/npcap-1.79.exe

**Comment by HenkPoley** [comment]
Current Ncap 1.81 fails to install with a message saying:

```plaintext
---------------------------
Npcap 1.81 Setup
---------------------------
Silent installation is only supported in Npcap OEM - https://npcap.com/oem/
---------------------------
OK   
---------------------------
```

I guess it will need a AutoHotkey similar solution as Nmap.

https://github.com/microsoft/winget-pkgs/tree/25342e6608f580032633d8d2304dc14eda21b1d0/manifests/i/Insecure/Npcap/1.81
https://github.com/microsoft/winget-pkgs/tree/master/manifests/i/Insecure/Npcap/1.81

**Comment by HenkPoley** [comment]
Possibly also update the `License:` line to https://npcap.com/oem/

https://github.com/microsoft/winget-pkgs/blob/25342e6608f580032633d8d2304dc14eda21b1d0/manifests/i/Insecure/Npcap/1.81/Insecure.Npcap.locale.en-US.yaml#L12

**Comment by aisgbnok** [comment]
FYI. From the Chocolatey Npcap package:

> The authors of npcap have requested that the software not be available on the Chocolatey Community Repository and I've unlisted all package versions:
>
> > Hello Paul and Chocolatey team,
> > 
> > I hope you are well. You may recall our discussions from July 2021 regarding the unwanted listing of our Npcap software on Chocolatey. While we respect that Chocolatey is a valuable platform for many users and software publishers, we still do not want Npcap on your platform. The main problem is that Chocolatey's "silent install" feature automatically clicks accept on the license screen without showing this critical legal information to the user. This screen notifies the user of important legal restrictions. For example they are generally only allowed to install up to 5 copies of the free demo version for themselves or their company. They need to buy our Nmap OEM product if they need more. That is how we fund the product.
> > 
> > In 2021, you hid the old version of Npcap on Chocolatey which was there at the time and you reported back that "no version of npcap is listed on the Chocolatey Community Repository" and "if any user goes there to look for npcap they will find the package with no versions to install and move on." We appreciated your prompt attention to the matter.
> > 
> > But it looks like you started the process of adding Npcap Version 1.8.1 a few weeks ago, and it is currently listed as "Waiting for Maintainer to take corrective action". It also looks like the copyrighted text from our Npcap web page has been republished verbatim without permission by Chocolatey. See the screenshot attached.
> > 
> > We want to reiterate our request that Chocolatey not include our Npcap software (any version) on your platform. If for some reason you have to keep an Npcap entry, we ask that you remove our copyrighted text and write your own summary like you had before.
> > 
> > Just let me know if you have any questions. Thanks for your cooperation.
> > 
> > Sincerely,
> > Gordon Lyon
> > Nmap Software LLC
>

Post from Paul Broadwith (Technical Engineering Manager Chocolatey). http://disq.us/p/32vbyol

TL;DR: Nmap Software relies on OEM license sales to fund Npcap's development, and allowing silent installation on the free/demo version would undermine the model that keeps it free for end users.

**Comment by Dragon1573** [comment]
✅ Confirmed.

<img width="1920" height="1288" alt="Image" src="https://github.com/user-attachments/assets/3049b14c-e5b7-48e4-87e0-6e8acd916ac2" />
<img width="958" height="619" alt="Image" src="https://github.com/user-attachments/assets/4d2788cc-1ac7-4fd2-9733-edbc56d5bedb" />
